src/components/colorPicker/ColorPicker.tsx

```diff
@@ -43,12 +44,18 @@ const ColorPicker = () => {
             className='w-5 h-5 rounded'
           ></div>
         </div>
-        <div
-          className='py-1.5 px-3.5 border border-slate-200 text-xs'
+        <StyledTextInput
           onClick={() => setShowPicker((prev) => !prev)}
+          value={appState?.appState.editorColor}
+          onChange={(e) => {
+            const hexColor = e.target.value.includes('#')
```

Comment:
@aatbip is it possible to add validation for hex color e.g.[https://codepen.io/dwu891/pen/zZxJLe]( https://codepen.io/dwu891/pen/zZxJLe)